### ISSUE TITLE
[Question] How to turn off incognito mode?

### ISSUE THREAD

**Issue by shivasagarrao** [dslul/openboard#249]
When I use [NewPipe](https://github.com/TeamNewPipe/NewPipe) app keyboard is in incognito mode. Is there a way to turn off incognito mode?

**Comment by mrx23dot** [comment]
What triggers incognito mode?
it seems to be enabled often, eg. typing into Searchbar ex, and duckduckgo,
I don't think it should be enabled unless being forced on by settings.

v1.4.5 on android12
(for reference gBoard doesn't have this issue)

**Comment by secretmango** [comment]
Its apps telling the keyboard "please use incognito mode". Signal has such a feature. I never used autocomplete, but it seems many apps do use this flst, that dont have a GUI to disable this.

So openboard needs to implement a setting "ignore incognito mode" and always have autocomplete.

**Comment by mrx23dot** [comment]
Yes please! 
"ignore incognito mode" feature

**Comment by derpafist** [comment]
It really gets annoying! Even when I use WhatsApp Status or Google Notes (yes, not entirely open source) it goes into incognito mode and not only does it stop learning but also correcting and suggesting words.

**Comment by mrx23dot** [comment]
It's fixed in the new app, it's called HeliBoard, based on openboard
https://github.com/Helium314/HeliBoard/issues/266#event-11588831011

**Comment by Hanif-F** [comment]
Yeah, I supposed I should checkout HeliBoard. Been in love with OpenBoard for the longest time, sad that for some reason it always have incognito on.
